Simplenote/Classes/SPTagsListViewController.m

```diff
@@ -635,17 +665,20 @@ - (NSArray *)sortDescriptors {
 }
 
 - (void)performFetch {
-    
     NSError *error = nil;
-	if (![self.fetchedResultsController performFetch:&error])
-    {
+	if (![self.fetchedResultsController performFetch:&error]) {
 	    NSLog(@"Unresolved error %@, %@", error, [error userInfo]);
         abort();
 	}
     
     [self.tableView reloadData];
 }
 
+- (void)refreshSortDescriptors {
```

Comment:
I'm gonna be a little pedantic with this one. ;)
Should this be named `refreshSortDescriptorsAndPerformFetch` to be more descriptive?  Or maybe the call to `performFetch` could be moved  to `tagsSortOrderWasUpdated` which seems to be the only caller of this method?